src/ggrc-client/js/components/unified-mapper/mapper-results-item-description.js

```diff
@@ -0,0 +1,60 @@
+/*
+ Copyright (C) 2020 Google Inc.
+ Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+ */
+
+import canStache from 'can-stache';
+import canDefineMap from 'can-define/map/map';
+import canComponent from 'can-component';
+import template from './templates/mapper-results-item-description.stache';
+
+const viewModel = canDefineMap.extend({
+  instance: {
+    value: null,
+  },
+  haveAdmin: {
+    get() {
+      return this.instance.access_control_list.length;
+    },
+  },
+  haveNotes: {
+    get() {
+      if (this.instance.notes !== undefined) {
+        return true;
+      } else {
+        return false;
+      }
+    },
+  },
+  hideBlock: {
+    get() {
+      return this.haveAdmin && this.haveNotes;
+    },
+  },
+  title: {
+    get() {
+      switch (this.instance.type) {
+        case 'TaskGroup': {
+          return 'Details';
+        }
+        case 'AssessmentTemplate': {
+          return 'Default assessment procedure';
+        }
+        default: {
+          return 'Description';
+        }
+      }
+    },
+  },
+  getDescription() {
```

Comment:
I suggest using attribute instead of function
```suggestion
   description: {
    get() {
      return this.instance.description
      || this.instance.procedure_description;
    },
  },
```